applications/crossbar/src/modules/cb_accounts.erl

```diff
@@ -1342,8 +1342,7 @@ set_notification_preference(Context, Preference) ->
 
 -spec create_account_mod(ne_binary()) -> any().
 create_account_mod(AccountId) ->
-    Db = kz_util:format_account_mod_id(AccountId),
-    kazoo_modb:create(Db).
+    kazoo_modb:create(kazoo_modb:get_db(AccountId)).
```

Comment:
wht_util had an an error that was fixed, the problem was not here, kz_services_modb handled the modb formats.

this pr is about returning a proper message in case account creation fails, lets keep the scope to that. 

to keep consistence, you can create another pr to remove usage of format_account_mod_id in all base code and deprecate it for 4.1 (add a note in announcements).